docker/nuclei/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM projectdiscovery/nuclei as builder
+
+FROM demisto/python3:3.8.6.12176
+
+COPY requirements.txt .
+
+RUN mkdir /app
+
+RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
+  && git clone https://github.com/projectdiscovery/nuclei-templates.git /app/nuclei-templates \
```

Comment:
@rotemreiss Can we do here a shallow clone to keep the image size small? Also do you want to always work on master? Is master always stable or is there need to work on a specific tag?

Reply:
Maybe also delete /app/nuclei-templates/.git after the clone?

Reply:
Actually, I will need to check it, since I think that on the latest version of Nuclei, they are already fetching the templates automatically and this clone can completely be removed.
I'll share an update asap.

Reply:
@rotemreiss you never got back on this. Is this still relevant?